plugins/janus_videoroom.c

```diff
@@ -2321,6 +2330,43 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	}
 }
 
+static void janus_videoroom_sdp_v_format_vp8(char *mline, int mline_size, int pt, int b, const char *video_mode) {
```

Comment:
That particular feature is pretty old and available everywhere c89 is.
(you're right that some format replacement types are gcc-specific or relatively new)